#!/usr/bin/env python
PACKAGE = "ema_stim_matrix"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# mygroup = gen.add_group("MyGroup", type="tab")  # types: 'hide', 'collapse', 'apply', 'tab'

gen.add("current",		int_t,  0, "Stimulator current", 0, 0, 80)
gen.add("pulse_width",	int_t,  0, "Stimulator current", 500, 0, 500)

exit(gen.generate(PACKAGE, "server", "DynamicParams"))

